src/webui/www/private/scripts/client.js

```diff
@@ -473,9 +508,9 @@ window.addEvent('load', function() {
             updateAltSpeedIcon(alternativeSpeedLimits);
         }
 
-        syncMainDataTimerPeriod = serverState.refresh_interval;
-        if (syncMainDataTimerPeriod < 500)
-            syncMainDataTimerPeriod = 500;
+        serverSyncMainDataInterval = serverState.refresh_interval;
```

Comment:
I know it is old code, you could do
`serverSyncMainDataInterval = Math.max(serverState.refresh_interval, 500);`